src/components/ProjectCard.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from "styled-components";
+
+import ProjectDefaultImage from '../images/ProjectDefaultImage.svg'
+import '../styles/components/ProjectCard.css';
+
+interface ProjectProps{
+    name: string,
+    presentation_image: string,
+    presentation_image_gradient_color: string,
+    description: string,
+    background_color: string,
+    in_progress: boolean
+}
+
+export default function ProjectCard(props: ProjectProps){
+
+    const ProjectCard = styled.div`
```

Comment:
Shouldn't CSS be in a .css file?

Reply:
Theoretically yep, but not, at same time. 
I don't know exactly if put it into a .css file is the correct pattern too, in this case. Here I'm declarating a component, a div with a specific style. So It theoretically could be on a css file, but that's the way that I saw people using this library.